.github/workflows/int-test-workflow.yml

```diff
@@ -5,6 +5,7 @@ on:
       - develop
       - master
       - feature-appframework
+      - CSPL-1604
```

Comment:
This is for testing this PR only and not to be merge eventually, isn't it?

Reply:
Yes @jambrosiano